news/447.fix

```diff
@@ -0,0 +1 @@
+- Add missing translate icon for the menu [szakitibi]
```

Comment:
The filename should be `447.bugfix` otherwise `towncrier` will not pick it up when generating the changelog entries for `CHANGES.rst` 😄